README.md

```diff
@@ -2,28 +2,35 @@
 
 This is the repo of the [FESTIM V&V report](https://festim-vv-report.readthedocs.io/) built with [Jupyter-book](https://jupyterbook.org/).
 
-
 ## Build locally
 
 First clone the repository:
 
-```
+```bash
 git clone https://github.com/festim-dev/V-V-report
 ```
 
 Create the correct conda environment with the required dependencies:
 
-```
+```bash
 conda env create -f environment.yml
 conda activate vv-festim-report-env
 ```
 
 You can then build the book with:
-```
+
+```bash
 jupyter-book build report
 ```
 
 To force the rebuild of all pages:
-```
+
+```bash
 jupyter-book build report --all
 ```
+
+## Contributing
```

Comment:
I wouldn't call this section contributing since it's not really about contribution but only how to convert markdown files to ipynb and vice versa